.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
+
+defaults: &defaults
+  docker:
+    - image: circleci/node:10-browsers
+  working_directory: ~/styledown
+
+
+jobs:
+  checkout_code:
+    <<: *defaults
+    steps:
+      - checkout
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+
+  install_dependencies:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: .
+      - restore_cache:
+          keys:
+            - styledown-node10-v1-{{ checksum "yarn.lock" }}
+      - run:
+          name: Avoid hosts unknown for github
+          command: mkdir ~/.ssh/ && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config
+      - run:
+          name: Yarn Install
+          command: yarn install --non-interactive --pure-lockfile
+      - save_cache:
+          key: styledown-node10-v1-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/styledown/node_modules
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+
+  # Uncomment after setting up eslint
+  # lint:
+  #   <<: *defaults
+  #   steps:
+  #     - attach_workspace:
+  #         at: .
+  #     - run:
+  #         name: Lint JavaScript
+  #         command: npm run lint:js
+
+  tests:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Run Tests
+          command: npm run test
+
+workflows:
+  version: 2
+  default:
+    jobs:
+      - checkout_code
+      - install_dependencies:
+          requires:
+            - checkout_code
+      - tests:
+          requires:
+            - install_dependencies
```

Comment:
end of file blank line missing
we should add an editor config in this project

Reply:
Yes, the project also need to be linted.
Feel free to open a PR for the editor config